systemd/crc-dnsmasq.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CRC Unit for configuring dnsmasq
+Requires=ovs-configuration.service
+After=ovs-configuration.service
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/bash -c "/usr/bin/ping -c1 gateway && echo 1 || echo 0"
```

Comment:
What is this condition for? user mode networking?

Reply:
yes, when in user-mode networking `gateway` is resolvable from inside the VM, so when ping returns success it means its user-mode networking